### PR TITLE
CRAN builds with gfortran 6.1 on R 3.4

### PR DESCRIPTION
**Pull request by jeroen**
@jimhester

### CODE REVIEW

**Comment by jeroen** [comment]
Actually this is wrong, the fortran file is a `dmg` not a `pkg` :(

**Comment by jeroen** [comment]
Update: added `hdutil` to extract the dmg file.

File: lib/travis/build/script/r.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ class R < Script
           # Heavy dependencies
           pandoc: true,
           latex: true,
+          fortran: false,
```
Comment (jimhester) [inline lib/travis/build/script/r.rb]: If we have an option for this, it definitely has to default to `true`. It also should work for non MacOS builds as well (just excluding `gfortran` from https://github.com/travis-ci/travis-build/pull/1193/files#diff-5b7e1aa01f1c6f058df6e0b9e2486c62R93 would do it I think)

**Comment by jeroen** [comment]
Thanks for the feedback. PR has been updated to default to `fortran:true` and also respect this option on Linux.

**Comment by jimhester** [comment]
LGTM, @BanzaiMan can you do the official review?

**Pull request merged**